pkg/asset/ignition/bootstrap/common.go

```diff
@@ -180,7 +180,7 @@ func (a *Common) generateConfig(dependencies asset.Parents, templateData *bootst
 	if err := AddSystemdUnits(a.Config, "bootstrap/systemd/common/units", templateData, commonEnabledServices); err != nil {
 		return err
 	}
-	if !templateData.IsOKD {
+	if !templateData.IsOKD || templateData.Invoker != "agent-installer" {
```

Comment:
Should it just be this:?
```suggestion
	if templateData.Invoker != "agent-installer" {
```

Reply:
no actually, nevermind.